### PR TITLE
fix(sdk-core): fix parameter misordering for user gpg key

### PR DESCRIPTION
**Pull request by rlutzbitgo**
TICKET: BG-00000

### CODE REVIEW

**Review by zahin-mohammad — DISMISSED** [review]

**Review by wesgraham — APPROVED** [review]

**Review by olibeyene — DISMISSED** [review]

**Comment by rlutzbitgo** [comment]
We did fix WP to make this still work on SDK 18.0.1, but we should still fix this bug here. WP will handle both cases.

File: modules/sdk-core/src/bitgo/utils/tss/ecdsa/ecdsa.ts [hunk 1]
```diff
@@ -697,8 +697,8 @@ export class EcdsaUtils extends baseTSSUtils<KeyShare> {
       encryptedSignerShare,
       vssProof,
       privateShareProof,
-      userPublicGpgKey,
-      publicShare
+      publicShare,
+      userPublicGpgKey
```
Comment (sachushaji) [inline modules/sdk-core/src/bitgo/utils/tss/ecdsa/ecdsa.ts:701]: Can you add a testcase to test this fix?,

Comment (zahin-mohammad) [reply modules/sdk-core/src/bitgo/utils/tss/ecdsa/ecdsa.ts:701]: The API will support both cases to be backwards compatible with the previous SDK version that was already released.

**Review by zahin-mohammad — APPROVED** [review]

**Comment by rlutzbitgo** [comment]
Feel free to merge this once it gets developer-relations-team approval

**Review by sachushaji — APPROVED** [review]

**Pull request merged**